internal/rbd/rbd_util.go

```diff
@@ -175,6 +189,15 @@ type imageFeature struct {
 	dependsOn []string
 }
 
+// migrationHandle is a struct which consists of required fields of a rbd volume
+// from migrated volume handle.
+type migrationHandle struct {
+	Version   string
+	Monitors  string
+	ImageName string
+	PoolName  string
```

Comment:
any specific reason to export these fields in an unexported structure?

Reply:
unexported @Madhu-1 .. done. thanks.